### ISSUE TITLE
Github Pages of the examples?

### ISSUE THREAD

**Issue by jacobhenke** [KDawg/YourFutureHTML#1]
It would be awesome if we could see the examples live on [Github pages](https://pages.github.com) (or somewhere else?).

**Comment by KDawg** [comment]
I agree with you. I gotta learn more about GutHub pages seeing what it offers us.
